server/src/main/java/org/elasticsearch/index/mapper/AbstractGeometryFieldMapper.java

```diff
@@ -72,9 +72,7 @@
         /**
          * Parses the given value, then formats it according to the 'format' string.
          *
-         * By default, this method simply parses the value using {@link Parser#parse}, then formats
-         * it with {@link Parser#format}. However some {@link Parser} implementations override this
```

Comment:
It's too bad that we removed this, it's nice to avoid parsing potentially very complex shapes when not necessary. I guess we can always revisit it as a performance optimization.